src/fs/managed.lua

```diff
@@ -32,18 +32,74 @@ do
 
   -- take the attribute file data and return a table
   local function load_attributes(data)
+    local attributes = {}
+    for line in data:gmatch("[^\n]+") do
+      local key, val = line:match("^(.-):(.+)$")
+      attributes[key] = tonumber(val)
```

Comment:
Is it possible that a corrupted attribute file could cause major errors? Since if a key is missing it'd be nil.

Reply:
Possibly - I can add some sanity checks / field defaults.